pkgs/shells/marcel/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, dill
+, psutil
+, makeWrapper
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "marcel";
+  version = "0.17.0-unstable-2023-11-11";
```

```suggestion
  version = "unstable-2023-11-11";
```
any reason we can't use the stable version though?

---

Using stable version now.